src/header/js/header.js

```diff
@@ -74,17 +74,19 @@ export default (data) => {
 
 	$header.addEventListener('header:update', ({detail}) => {
 		if (detail.brand) {
-			if (detail.brand.topStripe) {
-				$dispatch($brandStripe, 'header:update:brand', detail.brand);
-				$header.style.marginTop = '3px';
-			}
-			if (detail.brand.editTitle) {
-				$dispatch($inlineTitle, 'header:update:inlineTitle', detail.brand);
-			} else {
-				$dispatch($brand, 'header:update:brand', detail.brand);
-			}
+			detail.brand.root = true;
 		}
 
+		$dispatch($brandStripe, 'header:update:brand', detail.brand);
```

Comment:
These dispatches now fire even if the detail isn't present. The components here will handle an `undefined` detail value and hide their element.